advanced/button-interrupt/solution/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "button-interrupt"
+version = "0.1.0"
+authors = ["Anatol Ulrich <anatol.ulrich@ferrous-systems.com>"]
```

Comment:
We should either add you here, or change it to "Ferrous Systems GmbH" and leave out the e-mail address.